src/formatter/formatCommaPositions.ts

```diff
@@ -52,7 +52,7 @@ function groupCommaDelimitedLines(lines: string[]): string[][] {
     // when line ends with comma,
     // gather together all following lines that also end with comma,
     // plus one (which doesn't end with comma)
-    while (lines[i].match(/.*,$/)) {
+    while (lines[i].match(/.*,((\s*--.+)|$)/)) {
```

Comment:
For some reason `.+` is used here instead of `.*`. This means the regex won't match empty comments.

Also, it has an extra pair of parenthesis.

Reply:
Thanks, I'll add fixes